src/lib.rs

```diff
@@ -124,28 +143,35 @@ fn local_multistore_generate_piece_commitment_bytes_from_source<H: Hasher, R: Si
     Ok(comm_p_bytes)
 }
 
-fn padded<R: Sized + io::Read>(inp: &mut R, size: u64) -> PadReader<&mut R> {
+fn base2_padded<R: Sized + io::Read>(inp: &mut R, size: u64) -> Base2PadReader<&mut R> {
     let padded_size = padded_size(size);
 
-    let pad_reader = PadReader {
+    let base2_pad_reader = Base2PadReader {
         size: usize::try_from(size).unwrap(),
         padsize: usize::try_from(padded_size).unwrap(),
         pos: 0,
         inp: inp,
     };
 
-    pad_reader
+    base2_pad_reader
 }
 
+/*
+ * How you'd do it through filecoin_proofs::generate_piece_commitment()
+ * which would be the more "correct" way to call in because most things
+ * are done for you.
+ */
 #[allow(dead_code)]
```

Comment:
Please remove those `#[allow(dead_code)]` attributes, they are quite misleading (that's also OK to be done in a separate commit).